src/rpc/protocol/payloads.rs

```diff
@@ -448,11 +454,37 @@ impl ProtoDecoder for ObTableOperationRequest {
     }
 }
 
+/// Option for [`RawObTableOperation`]
+#[derive(Debug, Clone, PartialEq)]
+pub struct RawObTableOperationFlag {
+    // whether a check_and_execute option
+    pub check_and_execute: bool,
+    // check whether any data meet the filter in check_and_execute
+    pub check_exist: bool,
```

Comment:
```suggestion
    /// Whether a check_and_execute option
    pub check_and_execute: bool,
    /// check whether any data meet the filter in check_and_execute
    pub check_exists: bool,
```

BTW, the comment
> check whether any data meet the filter in check_and_execute

The `check_and_execute` seems just a boolean, where is `the filter` in it?